test/LaunchDarkly.JsonStream.Tests/StringTokenTest.cs

```diff
@@ -69,12 +70,24 @@ public void ParseLong(long n)
         [InlineData(-1234567890.5)]
         public void ParseDouble(double n)
         {
-            var s = n.ToString();
+            var s = n.ToString(CultureInfo.InvariantCulture);
```

Comment:
The test should also work regardless of where it is being ran. Changing the cultures in the other test exposed this. I am not sure about resetting them.

Reply:
(In that nothing SHOULD depend on it being reset).